src/main/java/org/sonarlint/intellij/ui/nodes/IssueNode.java

```diff
@@ -28,14 +28,15 @@
 import java.io.IOException;
 import javax.annotation.Nonnull;
 import org.sonarlint.intellij.issue.IssuePointer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonarlint.intellij.issue.IssuePointer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)